src/ai/ai.controller.ts

```
@@ -0,0 +1,80 @@
+    private readonly gameService: GameService,
+  ) {}
+
+  /**
+   * Coach suggestion: does NOT modify game state.
+   * If fen not provided, loads the game and uses its current FEN.
+   */
+  @Post(':id/suggest')
+  async suggest(
+    @Param('id') gameId: string,
+    @Body() body: SuggestBodyDto,
+  ) {
```

ESLint

[error] Replace `⏎····@Param('id')·gameId:·string,⏎····@Body()·body:·SuggestBodyDto,⏎··` with `@Param('id')·gameId:·string,·@Body()·body:·SuggestBodyDto`

(prettier/prettier)